src/random.h

```diff
@@ -28,26 +28,23 @@ uint256 GetRandHash();
 void GetStrongRandBytes(unsigned char* buf, int num);
 
 /**
- * Seed insecure_rand using the random pool.
- * @param Deterministic Use a deterministic seed
+ * Fast randomness source. This is seeded once with secure random data, but
+ * is completely deterministic and insecure after that.
+ * This class is not thread-safe.
  */
-void seed_insecure_rand(bool fDeterministic = false);
+class FastRandomContext {
```

Comment:
`FastRandomContext` is basically a clone of `InsecureRand` which was implemented in Dash earlier. I think it makes sense to s/InsecureRand/FastRandomContext/ elsewhere in code and drop `InsecureRand` in `random.h/cpp` now.